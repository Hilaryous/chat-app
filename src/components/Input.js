// @flow
import styled from 'react-emotion'

export default styled('input')({
  padding: '16px',
  border: 'none',
  background: '#FBF6FE',
})
