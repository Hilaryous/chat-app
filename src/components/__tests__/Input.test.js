import React from 'react'
import renderer from 'react-test-renderer'
import Input from '../Input'

describe('Input', () => {
  it('shows the correct snapshot tree for the Input component', () => {
    const tree = renderer.create(
      <Input
        type="text"
        placeholder="Message"
        value="hi"
        onChange={() => {}}
      />,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
