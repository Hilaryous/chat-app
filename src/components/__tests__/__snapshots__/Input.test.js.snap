// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input shows the correct snapshot tree for the Input component 1`] = `
<input
  className="css-zof107"
  onChange={[Function]}
  placeholder="Message"
  type="text"
  value="hi"
/>
`;
